Mlem/Views/Shared/Links/Community Link View.swift

```diff
@@ -9,6 +9,26 @@ import Foundation
 import SwiftUI
 import CachedAsyncImage
 
+func shouldClipAvatar(community: APICommunity) -> Bool {
+    let clipOptOut = ["beehaw.org"]
```

Comment:
Can we consolidate these lists as they are going to be the same right?

Reply:
oh yea
Fixed

I had made it internal before I realized I needed a different function for the sidebar.